frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsSection.tsx

```diff
@@ -1,181 +1,64 @@
 
 import { FormAuthType, FormSettings, FormStatus } from '~shared/types/form'
 
-import { GUIDE_SPCP_ESRVCID } from '~constants/links'
-import InlineMessage from '~components/InlineMessage'
-import Link from '~components/Link'
-import Radio from '~components/Radio'
-import { Tag } from '~components/Tag'
+import Toggle from '~components/Toggle'
 
 import { useAdminForm } from '~features/admin-form/common/queries'
 import { isMyInfo } from '~features/myinfo/utils'
 
-import { useMutateFormSettings } from '../../mutations'
-
-import { FORM_AUTHTYPES } from './constants'
-import { EsrvcIdBox } from './EsrvcIdBox'
-
-const esrvcidRequired = (authType: FormAuthType) => {
-  switch (authType) {
-    case FormAuthType.SP:
-    case FormAuthType.MyInfo:
-    case FormAuthType.CP:
-      return true
-    default:
-      return false
-  }
-}
+import { AuthSettingsDescriptionText } from './AuthSettingsDescriptionText'
+import { AuthSettingsDisabledExplanationText } from './AuthSettingsDisabledExplanationText'
+import { AuthSettingsSingpassSection } from './AuthSettingsSingpassSection'
+import { FormSingpassAuthToggle } from './FormSingpassAuthToggle'
 
 interface AuthSettingsSectionProps {
   settings: FormSettings
 }
 
 export const AuthSettingsSectionSkeleton = (): JSX.Element => {
   return (
-    <Radio.RadioGroup>
-      {Object.entries(FORM_AUTHTYPES).map(([authType, textToRender]) => (
-        <Radio isDisabled key={authType}>
-          <Skeleton>{textToRender}</Skeleton>
-        </Radio>
-      ))}
-    </Radio.RadioGroup>
+    <Skeleton>
+      <AuthSettingsDescriptionText />
+      <Toggle description="" label="" />
+    </Skeleton>
   )
 }
 
 export const AuthSettingsSection = ({
   settings,
 }: AuthSettingsSectionProps): JSX.Element => {
-  const { mutateFormAuthType } = useMutateFormSettings()
   const { data: form } = useAdminForm()
 
   const containsMyInfoFields = useMemo(
     () => form?.form_fields.some(isMyInfo) ?? false,
     [form?.form_fields],
   )
 
-  const [focusedValue, setFocusedValue] = useState<FormAuthType>()
-
-  const isFormPublic = settings.status === FormStatus.Public
-
-  const isDisabled = useCallback(
-    (authType: FormAuthType) =>
-      isFormPublic || containsMyInfoFields || mutateFormAuthType.isLoading,
-    [isFormPublic, containsMyInfoFields, mutateFormAuthType.isLoading],
-  )
-
-  const isEsrvcIdBoxDisabled = useMemo(
-    () => isFormPublic || mutateFormAuthType.isLoading,
-    [isFormPublic, mutateFormAuthType.isLoading],
-  )
-
-  const handleEnterKeyDown: KeyboardEventHandler = useCallback(
-    (e) => {
-      if (
-        (e.key === 'Enter' || e.key === ' ') &&
-        focusedValue &&
-        !isDisabled(focusedValue) &&
-        focusedValue !== settings.authType
-      ) {
-        return mutateFormAuthType.mutate(focusedValue)
-      }
-    },
-    [focusedValue, isDisabled, mutateFormAuthType, settings.authType],
+  const isFormPublic = useMemo(
```

Comment:
Very trivial comparison, [avoid](https://www.notion.so/opengov/You-might-not-need-useMemo-95dcc0f2f2e544eabc1ff1a5f2e157d0?pvs=4) using `useMemo`.

The result is a boolean which is trivial. Very similar to another [case](https://github.com/opengovsg/FormSG/pull/7350#discussion_r1625672180) I reviewed 😆

Reply:
removed and thanks for the explanation on when to use useMemo!